public/js/core.js

```diff
@@ -37,12 +37,14 @@ app.controller('mainController', function ($scope, $state, Todo) {
 
     $scope.createTodo = function () {
         $scope.loading = true;
-
-        if ($scope.newTodo.text !== '') {
+        if ($scope.newTodo.text !== "") {
```

Comment:
I prefer write more linear code - for example here we can return value (exit from function) if 
```javascript 
($scope.newTodo.text === "")
```